content/en/agent/kubernetes/tag.md

```diff
@@ -31,10 +31,10 @@ The Agent can autodiscover and attach tags to all data emitted by the entire pod
   | `display_container_name`      | High         | Pod status                                                              | N/A                                                 |
   | `pod_name`                    | Orchestrator | Pod metadata                                                            | N/A                                                 |
   | `oshift_deployment`           | Orchestrator | Pod annotation `openshift.io/deployment.name`                           | OpenShift environment and pod annotation must exist |
-  | `kube_ownerref_name`          | Orchestrator | Pod ownerref                                                            | Pod must have an owner                              |
-  | `kube_job`                    | Orchestrator | Pod ownerref                                                            | Pod must be attached to a job                       |
-  | `kube_replica_set`            | Orchestrator | Pod ownerref                                                            | Pod must be attached to a replica set               |
-  | `kube_service`                | Orchestrator | Kubernetes service discovery                                            | Pod is behind a Kubernetes service                  |
+  | `kube_ownerref_name`          | Low          | Pod ownerref                                                            | Pod must have an owner                              |
+  | `kube_job`                    | Low          | Pod ownerref                                                            | Pod must be attached to a job                       |
```

Comment:
This one is Orchestrator when the job is part of a cron job, Low otherwise https://github.com/DataDog/datadog-agent/blob/main/pkg/tagger/collectors/workloadmeta_extract.go#L390-L395